InvoiceMaker/Views/Clients/Create.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@model InvoiceMaker.FormModels.CreateClient
+
+<h1>Create a client</h1>
+
+
+@using (Html.BeginForm("Create", "Clients", FormMethod.Post, new { @class = "pure-form pure-form-aligned" }))
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.